### DISCUSSION TITLE
bookmark issue

### DISCUSSION THREAD

**Discussion by Qoqex** [3kh0/ext-remover#525] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hi I was planning on using this but then realized that I can't do it since My school blocked me from importing bookmarks is there a way to bypass that? and add bookmarks either ways???

**Comment by Aka-but-nice** [comment]
> Hi I was planning on using this but then realized that I can't do it since My school blocked me from importing bookmarks is there a way to bypass that? and add bookmarks either ways???

Could you explain more? Maybe send a picture or video of what is going on, if you actually mean "importing" in the actual sense

**Comment by obkduishinijf** [comment, marked as answer]
blud said importing bookmarklets

**Reply by Aka-but-nice** [reply]
💀

**Comment by Qoqex** [comment]
![image (1)](https://user-images.githubusercontent.com/74578636/227070043-6e1bb909-e796-4846-b090-4482c919b775.png)
Like the option to allow bookmarks isnt allowed since my school admin has it blocked.
